src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,5 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
only box-sizing can be in this block `*{}`

Reply:
Could you explain, please, why only box-sizing? I want to reset all margins and paddings